frontend/js/src/home/Homepage.tsx

```diff
@@ -248,11 +293,196 @@ function HomePage() {
           </div>
         </div>
+        <FontAwesomeIcon
+          icon={faSortDown}
+          className="homepage-arrow"
+          size="3x"
+          onClick={() => {
+            homepageLowerInfoRef.current?.scrollIntoView({
+              behavior: "smooth",
+              block: "start",
+            });
+          }}
+        />
+      </div>
+
+      <div className="homepage-information" ref={homepageLowerInfoRef}>
+        <FontAwesomeIcon
+          icon={faSortUp}
+          className="homepage-arrow"
+          size="3x"
+          onClick={() => {
+            homepageLowerRef.current?.scrollIntoView({
+              behavior: "smooth",
+              block: "start",
+            });
+          }}
+        />
+
+        <Blob
+          width={200}
+          height={200}
+          randomness={1.5}
+          className="homepage-information-vector-1"
+          style={{ animationDelay: "-10s" }}
+        />
+
+        <div className="homepage-information-grey-box" />
+
+        <div className="homepage-information-content">
+          <div className="info-column">
+            <div className="info-icon">
+              <img
+                src="/static/img/homepage/LB-Open-Source-Info.png"
+                alt="Open source"
+              />
+            </div>
+            <h2>Open source</h2>
+            <p>
+              &quot;Open source is source code that is made freely available for
+              possible modification and redistribution... A main principle of
+              open source software development is peer production, with products
+              such as source code, blueprints, and documentation freely
+              available to the public.&quot; -{" "}
+              <Link
+                to="https://en.wikipedia.org/wiki/Open_source"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                Wikipedia
+              </Link>
+            </p>
+            <p>
+              The ListenBrainz website, the MusicBrainz database, and the music
+              and listen data that our users contribute is all open source. This
+              means that anyone can view the code, contribute improvements and
+              new features, and copy and modify it for their own use.
+            </p>
+            <p>
+              Thousands of people contribute code or data to ListenBrainz and
+              its sister projects for no monetary return. Contributors do this
+              to further humanity&apos;s global store of knowledge, and to
+              safeguard and share data about the artists and music that we love.
+            </p>
+            <p>
+              Data wants to be free, and we believe that by sharing what we
+              have, rather than jealously guarding it for the benefit of a few,
+              everyone can benefit.
+            </p>
+            <Link
+              to="https://github.com/orgs/metabrainz/repositories"
```

Comment:
I think this page is a better overview, with pinned repos and more:
```suggestion
              to="https://github.com/orgs/metabrainz"
```